plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -23,26 +24,26 @@ const config = {
 		[
 			'html',
 			{
-				outputFolder: 'output',
-				open: CI ? 'never' : 'always',
+				outputFolder: PLAYWRIGHT_HTML_REPORT ?? './test-results/playwright-report',
+				open: CI ? 'never' : 'on-failure',
```

Comment:
Great catch @alopezari. That isn't an intended change. I'll push a commit to revert that. That change ended up there because that's just how I personally preferred the Playwright reports to be configured. 😅

Reply:
> I spotted that my test-results/report folder is empty for both api-core-tests and e2e-pw. Is that normal?

Yep, it is. It only gets populated with files (screenshots, videos, and trace files) when there were failures encountered, since we configured Playwright to only generate them on failures.

The `report` folder would look like this if there were failures:
<img width="586" alt="T3iZQcHxeS" src="https://user-images.githubusercontent.com/4509348/207849029-3404a46b-2a6c-4d4a-b139-3124836d8cb1.png">

Reply:
Great, thanks for confirming! 🙌

Reply:
Already reverted to 'always'

Reply:
Thanks, @rodelgc!

The only thing pending would be the issue that @roykho mentioned [here](https://github.com/woocommerce/woocommerce/pull/35907#pullrequestreview-1214195533) (I got the same result).

I think once that's solved, we can approve this PR.